exiftool.go

```diff
@@ -84,16 +85,30 @@ func NewExiftool(opts ...func(*Exiftool) error) (*Exiftool, error) {
 	if err = e.cmd.Start(); err != nil {
 		return nil, fmt.Errorf("error when executing command: %w", err)
 	}
+	e.log("executing shell command:\n\t", e.cmd.String())
 
 	return &e, nil
 }
 
+func (e *Exiftool) log(args ...interface{}) {
+	for _, w := range e.debugWriters {
+		fmt.Fprintln(w, args...)
```

Comment:
Error should be handled (`fmt.Fprintln`)

Reply:
Ah, ya, I didn't think about error handling for this. I'm thinking just pass it back, and in `ExtractMetadata()`, we can let it silently fail (since it doesn't have error in the return signature). How does that sound?

Reply:
also, how do you feel about `github.com/pkg/errors`? It allows adding stack traces to your errors which makes for much simpler troubleshooting. I use it in all my projects now. No worries if you'd rather stick with the standard library.

Reply:
I pushed a commit with the error handling (following my suggestion above). Let me know what you think.

Reply:
> I pushed a commit with the error handling (following my suggestion above). Let me know what you think.

That's perfect !
I'd rather like to stick with the standard library (I'm thinking about also dropping testify).